lib/hanami/hanamirc.rb

```diff
@@ -119,12 +118,21 @@ def exists?
 
     private
 
+    def parse(str)
+      {}.tap do |hash|
+        str.split("\n").each do |line|
+          key, value = line.split('=')
```

Comment:
I working on Parser module, because `.hanamirc` can stores comments, empty lines and so on